canonical/canonical.com
# [WD-9668] remove one tab and change content on /solutions/telco

## Done

Copy update according to the [copy doc](https://docs.google.com/document/d/1yRSYYY8_UNfZzdLcp_fYxi-STGm40o3UP9vD2vW6brw/edit) (please refer to suggestions from Gloria from March 15th).

## QA

- Navigate to [/solutions/telco](https://canonical-com-1221.demos.haus/solutions/telco) and check the changes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9668
